cnosdb/src/main/java/cn/edu/tsinghua/iot/benchmark/cnosdb/CnosConnection.java

```diff
@@ -0,0 +1,32 @@
+package cn.edu.tsinghua.iot.benchmark.cnosdb;
+
+import okhttp3.*;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.concurrent.TimeUnit;
+
+public class CnosConnection {
+  private final String url;
+
+  OkHttpClient client;
+
+  CnosConnection(String urlString, String cnosDbName) throws MalformedURLException {
+    ConnectionPool connectionPool = new ConnectionPool(100, 5, TimeUnit.MINUTES);
```

Comment:
Perhaps the number of concurrency can be controlled to some extent by BM's CLIENT_NUMBER parameter